examples/treeple/treeple_tutorial_CMI.py

```diff
@@ -1,18 +1,24 @@
 """
-====================================
-Treeple tutorial for calculating CMI
-====================================
+=====================
+2-1b: Calculating CMI
```

Comment:
I don't think most ppl will know the order of our figures. Can we remove the numbering in the title? We can still order the sphinx examples as how it shows up tho.

Reply:
This is not related to the figures, just to separate single view and multiview. If we can order them with other methods, that would be great.